utils/CMakeLists.txt

```diff
@@ -1,2 +1,2 @@
-add_subdirectory(hlc)
+#add_subdirectory(hlc)
```

Comment:
@litghost I know we are planning to remove hlc, but this is premature. Looks like reenabling this causes build errors.